src/styles/blocks/main.scss

```diff
@@ -0,0 +1,112 @@
+@import "../utils/variables";
+
+.main__catalog {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  max-width: 944px;
+  margin: auto;
+  padding-top: 50px;
+}
+
+.card__code,
+.card_rating__reviews,
+.card__price-label {
+  font-size: 11px;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  width: 200px;
+  padding: 29px 14px 14px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4) {
+    margin-right: 0;
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    padding-top: 45px;
+    line-height: 15px;
+    font-size: 12px;
+    font-weight: 500;
+  }
+
+  &__code {
+    padding-top: 6px;
+    align-self: flex-start;
+  }
+
+  &_rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: inherit;
+    padding: 15px 15px 0;
+  }
+
+  &__price {
+    display: inherit;
+    justify-content: space-between;
+    align-items: flex-end;
+    width: inherit;
+    padding: 20px 15px;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__link-buy {
+    display: inherit;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-decoration: none;
+    background-color: $color-active;
+    border: 1px solid $color-active;
+
+    &:hover {
+      background-color: transparent;
+      color: $color-active;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url("../images/star.svg");
+    width: 15px;
+    height: 15px;
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+  }
+
+  &--4 &__star:nth-child(-n+4) {
```

Comment:
include all the modifiers here, so that you could change stars block modifier in index.html and number of active stars would change